### ISSUE TITLE
Open dialog appears behind the Fritzing window

### ISSUE THREAD

**Issue by bastianilso** [flathub/org.fritzing.Fritzing#5]
When I press "Files -> Open.." the open dialog appears behind the Fritzing window instead of in front of it.

**Comment by mMerlin** [comment]
What environment are you running?  I saw that on Fedora 27, but on Fedora 29 the dialog opened on top.  I **think** it is an issue with the host environment, not the application.

**Comment by bastianilso** [comment]
I was running Arch Linux. I just tested on Fedora 29 too and the dialog opened on top.

I wonder how can this be an issue with the host environment when Fritzing is flatpak'ed? Is it because the gtk file portal I had installed is out of date maybe?

**Comment by gasinvein** [comment]
Yes, it's likely to be caused by host environment, specifically `xdg-desktop-portal-gtk`. App inside the sandbox talks to the portal over dbus to explicitly gain access to host files and other features, so the filepicker window opens "outside" the sandbox.
